tests/unit/lms/services/lti_registration_test.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import sentinel
+
+import pytest
+
+from lms.services.lti_registration import LTIRegistrationService, factory
+from tests import factories
+
+
+class TestLTIRegistrationService:
+    def test_get(self, svc, registration):
+        assert svc.get(registration.issuer, registration.client_id) == registration
+
+    def test_factory(self, pyramid_request):
+        service = factory(sentinel.context, pyramid_request)
```

Comment:
Normally we make some assertions about how the service is constructed here using an autospecc'ed fixture. Which is a bit tiring, but is easy enough to churn out.